src/site/stages/build/drupal/graphql/bioPage.graphql.js

```diff
@@ -57,7 +57,7 @@ function getNodePersonProfilesSlice(operationName, offset, limit) {
       nodeQuery(
         limit: ${limit}
         offset: ${offset}
-        sort: { field: "changed", direction:  ASC }
+        sort: { field: "nid", direction:  ASC }
```

Comment:
Are these changes going to impact the cms export build as far as discrepancies?

Reply:
Sorry I missed this. No, unless the Transformers were relying on an exact array order from the raw, unsorted GraphQL response and I can't imagine that was the case?